opensrp-reveal/src/main/java/org/smartregister/reveal/fragment/SummaryFormsFragment.java

```diff
@@ -31,6 +31,20 @@
 
     private LocationUtils locationUtils;
 
+    private Button btnDailySummary;
+
+    private Button btnTeamLeaderDos;
+
+    private Button btnCbSprayArea;
+
+    private Button btnIrsSaDecision;
+
+    private Button btnMobilization;
+
+    private Button btnIrsFieldOfficer;
+
+    private Button btnVerificationForm;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'btnVerificationForm' could be replaced by a local variable.](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=5066421)